Improving Website

KommToby opened the pull request:
General cleanup and added a few points of internal commentary to support other people helping out with making the website. Hopefully some of the issues i've had with the imaging can be resolved by other people or when I learn enough about backgrounds to do it myself.

- Added a subtitle to the title of the webpage referencing the NSE placements of this year.
- Cleaned up the HTML code to all direct to the css file instead of styling within the HTML itself
- Added a NSE.png to reference when showing the page

KommToby committed:
Update index.css

General clean-up of the HTML converted to CSS, added background support with other main css references for different parts of the HTML

KommToby committed:
Update index.html

Added a subtitle to the Title mentioning our NSE placements
Also cleaned up the divs by referencing the CSS file instead of styling within the HTML itself

KommToby committed:
Add files via upload

NSE Logo

KommToby committed:
Addition of pages

Added the pages mentioned on the top left to link to

- About
- News
- Contact

KommToby committed:
Added links to other pages

KommToby committed:
Logos & Links

Added more content to the contact us page

- Added invite to our discord on the Discord logo
- Added link to our Twitter account on the Twitter logo
- Added link to our Facebook page on the Facebook logo
- Added link to our Instagram page on the Instagram logo
- Added link to make a new email to our email on the Gmail logo
- Added link to the University's website on the HWU Logo (Thanks Luke)

KommToby commented:
Been working a lot on the website, so i'd recommend having a brief view of all of the previous commits before looking at the final one before accepting it through.

I have an issue with the fact I want to change the main html file to home.html but I don't know how the domain we have works with the website, It seems like you originally set it up for all of the websites to be homed within one html file, but that's really inefficient and messy

So I was hoping there was a way for us to make the files orientated like
wattesports.co.uk/home
wattesports.co.uk/contact
etc
Instead of having everything under wattesports.co.uk (Which I assume is linked to the index.html file) and linking all of the other "pages" to other hidden content on the page

KommToby committed:
Added Twitter to News

- Added Twitter embed to @wattEsports to the newsfeed

Danial-Q commented on About.html:
```diff
@@ -0,0 +1,48 @@
+
+
+</head>
+<body> <! Can we add a transparent background of our logo? I've tried this but I couldn't seem to get the scaling correct>
```
`opacity`, `z-index` and just general sizing css properties should achieve this

KommToby replied:
Could you or someone else give it a bash? Using the background style of the body it seemed limited on what I could do it it (i.e. adding opacity) so I had to do that manually in photoshop, and then once that was imported, I couldn't get the logo to scale with the website correctly when the window was resized.

Danial-Q commented on About.html:
```diff
@@ -0,0 +1,48 @@
+			<a class="right" href="https://discord.wattesports.co.uk">Discord</a>
+		</div>
+		<div>
+			<a href=https://www.hw.ac.uk/> <img src="hw-logo.svg" alt="Heriot Watt Logo" width="170" style="float: right;padding: 20px 20px 20px 20px;"> </a>
```
Should consider making a folder for all images and importing from there just to keep stuff organized. Same goes for every new page added and such

KommToby replied:
So thinking just files called "images" and "pages" or something?

Can they still be called by just using src="filename.png" or do you have to reference the folder they're stored in too?

Danial-Q replied:
You'd have to reference the folder too but you can just use `..` to go up a level in the file tree so it's fairly simple to do

Danial-Q commented on Contact.html:
```diff
@@ -0,0 +1,53 @@
+</head>
+<body> <! Can we add a transparent background of our logo? I've tried this but I couldn't seem to get the scaling correct>
+	<header>
+		<div class="topnav"> 
```
Could use something like react or something to make each page more modular I.E you'd write a `<NavBar />` component then can import that to each page, as to not replicate code between pages. Makes maintenance much simpler too as you would only have to update one place if you decided to add another tile into the NavBar.

Mind react is a JS library, so y'all would to have to learn JS, but I'm more than willing to teach y'all if you want and it'd be a good skill to have.

Same thing could be achieved using `<iframe />`  too which is simpler but has it's risks associated with it

KommToby replied:
I did not do topnav at all, all of that system was In here before I started working on it, I think Kyle or Colin made the base website using some google tool, but i've used iframe before so maybe I can look into this.

Danial-Q commented on Contact.html:
```diff
@@ -0,0 +1,53 @@
+			<a class="right" href="https://discord.wattesports.co.uk">Discord</a>
+		</div>
+		<div>
+			<a href=https://www.hw.ac.uk/> <img src="hw-logo.svg" alt="Heriot Watt Logo" width="170" style="float: right;padding: 20px 20px 20px 20px;"> </a>
```
inline styling reeee

Simber1 replied:
I was having issues with the SVG so I inlined it.

KommToby committed:
Sorted Logos Into Logos Folder

Changed all the pages to support the logos in a new folder
Moved all the used logos into a separate folder for cleanup

KommToby committed:
Delete safari-pinned-tab.svg

KommToby committed:
Delete mstile-150x150.png

KommToby committed:
Delete hw-logo.svg

KommToby committed:
Delete gmail.png

KommToby committed:
Delete favicon.ico

KommToby committed:
Delete favicon-32x32.png

KommToby committed:
Delete favicon-16x16.png

KommToby committed:
Delete apple-touch-icon.png

KommToby committed:
Delete android-chrome-512x512.png

KommToby committed:
Delete android-chrome-192x192.png

KommToby committed:
Delete Twitter.png

KommToby committed:
Delete NSE.png

KommToby committed:
Delete Instagram.png

KommToby committed:
Delete Facebook.png

KommToby committed:
Delete Discord.png

Deleting all these one-by-one was awful and I hope to never have to do it again

kbourbery committed:
Update index.html

kbourbery committed:
Added teams button to all pages

kbourbery committed:
Replace "About" with "Teams"

Danial-Q commented on index.css:
```diff
@@ -52,6 +50,38 @@ p{
 
+/* Class for adding a background image - Not being used right now as I can't get it to work correctly */
+.background{
+	background: url(https://cdn.discordapp.com/attachments/698264458996351016/706052590294663238/Wattesportspartial.png);
```
Should have this as an img in the repo rather than a link that could eventually be removed

kbourbery committed:
Finished previous commit

KommToby committed:
Replaced web url with Image

Replaced the transparent background image link with a reference to the low opacity logo stored in "Logos"

kbourbery committed:
Updated info on Teams.html

KommToby committed:
Merge branch 'master' of https://github.com/KommToby/wattesports.co.uk

KommToby committed:
Merge branch 'master' of https://github.com/KommToby/wattesports.co.uk

kbourbery committed:
Fixed kyles shitty spelling

kbourbery committed:
Merge branch 'master' of https://github.com/KommToby/wattesports.co.uk

kbourbery committed:
Part 2

KommToby committed:
Started to add sponsors

- Added White Hive Logo
- Added Pink Union Logo
- Begun to input them into footer

KommToby committed:
Merge branch 'master' of https://github.com/KommToby/wattesports.co.uk

kbourbery committed:
Added Hive Logo to footer

kbourbery committed:
Fixed footer

-Aligned both logos to center of text

Danial-Q commented on News.html:
```diff
@@ -0,0 +1,48 @@
+			<a href=https://www.hw.ac.uk/> <img src="logos/hw-logo.svg" alt="Heriot Watt Logo" width="170" style="float: right;padding: 20px 20px 20px 20px;"> </a>
+		</div>
+			<div class="title">
+			<h1> News </h1>
```
Adding spaces here will add spaces to the title, it should be `<h1>News</h1>`

KommToby committed:
Merged both footer css

Danial-Q commented on Teams.html:
```diff
@@ -0,0 +1,48 @@
+	</header>
+	<main>
+		<div class="main">
+			<p> We currently field 11 teams across League of Legends, CS:GO, Overwatch, Rainbow Six: Siege, DOTA 2, and PUBGM, as well as a selection of solo players in Tekken, SSBU and FIFA.</p>
```
Spaces

KommToby committed:
Merge branch 'master' of https://github.com/KommToby/wattesports.co.uk

KommToby committed:
Removed Evil Spaces

KommToby committed:
Renamed Index to Home

renamed index.html to home.html for easier access in future

Simber1 reviewed (approved):
All looks good 👍
